### DISCUSSION TITLE
[Question] VIES button

### DISCUSSION THREAD

**Discussion by waran70** [YetiForceCompany/YetiForceCRM#13827] [Feature improvement]
No clue why VIES button shows while Save | Cancel action. Maybe better in-field bttn?

**Comment by mariuszkrzaczkowski** [comment]
show an example

**Comment by waran70** [comment]
![image](https://user-images.githubusercontent.com/6852348/100197027-52d9bd00-2f2c-11eb-9747-00ce485b956b.png)

Accounts module

**Comment by mariuszkrzaczkowski** [comment]
I meant an example of where you want to put it

**Comment by waran70** [comment]
![image](https://user-images.githubusercontent.com/6852348/100197864-8ec15200-2f2d-11eb-807e-97539ea4ee51.png)
